src/types.ts

```diff
@@ -421,6 +421,13 @@ export interface GetSwapQuoteResponse extends SwapQuoteResponsePartialTransactio
     estimatedGasTokenRefund: BigNumber;
     allowanceTarget?: string;
     quoteReport?: QuoteReport;
+    prices?: SourceComparison[];
+}
+
+export interface SourceComparison {
+    name: ERC20BridgeSource | '0x';
+    price: BigNumber | null;
```

Comment:
any strong reason to use nulls instead of undefined? 

if we used undefined we could do `price?: BigNumber;` instead of `price: BigNumber | null;` which feels more consistent w/ our other code

Reply:
makes sense 👍